PowerShell/JumpCloud Module/Public/Commands/Get-JCCommand.ps1

```diff
@@ -70,31 +95,139 @@ CommandID has an Alias of _id. This means you can leverage the PowerShell pipeli
 
                 $skip += $limit
                 Write-Debug "Setting skip to $skip"
-
-                $resultsArray += $results.results
-                $count = ($resultsArray).Count
+                $resultsArrayList += $results.results
+                $count = ($resultsArrayList).Count
                 Write-Debug "Results count equals $count"
             }
         }
 
-        elseif ($PSCmdlet.ParameterSetName -eq 'ByID')
-
+ 
+        switch ($PSCmdlet.ParameterSetName)
         {
-            foreach ($uid in $CommandID)
+            SearchFilter
+            {
+
+                while ((($resultsArrayList.Results).Count) -ge $Counter)
+                {
+
+                    if ($returnProperties)
```

Comment:
I don't see returnProperties in the available parameters, is this something that we wish to include? Get-JCUser and Get-JCSystem both have this functionality.

Example: [Get-JCUser ReturnProperties](https://github.com/TheJumpCloud/support/blob/master/PowerShell/JumpCloud%20Module/Public/Users/Get-JCUser.ps1#L76-L78)